crates/oxc_linter/src/rules/no_mixed_operators.rs

```diff
@@ -220,17 +220,15 @@ const PRECEDENCES: [u8; 27] = [
   3
 ];
 
-const ARITHMETIC: &[&str] = &OPERATORS[..6];
```

Comment:
error[E0277]: the trait bound `[&str; 27]: ~const std::ops::Index<RangeTo<usize>>` is not satisfied
   --> crates/oxc_linter/src/rules/no_mixed_operators.rs:223:30
    |
223 | const ARITHMETIC: &[&str] = &OPERATORS[..6];
    |                              ^^^^^^^^^^^^^^ `[&str; 27]` cannot be indexed by `RangeTo<usize>`
    |
    = help: the trait `~const std::ops::Index<RangeTo<usize>>` is not implemented for `[&str; 27]`
note: the trait `std::ops::Index<RangeTo<usize>>` is implemented for `[&str; 27]`, but that implementation is not `const`